generic/tests/netperf.py

```diff
@@ -653,6 +658,14 @@ def all_clients_up():
             return True
         return False
 
+    def stop_netperf_clients():
+        if params.get("os_type_client") == "linux":
+            ssh_cmd(clients[-1], params.get("client_kill_linux"),
+                    ignore_status=True)
+        else:
+            ssh_cmd(clients[-1], params.get("client_kill_windows"),
+                    ignore_status=True)
+
```

Comment:
If you use "ignore_status=True" here, how do you know whether the cmd is run successfully?

Reply:
no need to care if it runs successfully, even it fail, the threading.Thread will kill itself when netperf client is done , here kill command just make sure when effective tests are done while netperf client is still running, we need to force to kill netperf client for saving testing time then run next test earlier. 

for example, we runs 60s netperf test, but we only collect 45s data after all session start, if it takes 5s on all session start,  effective test takes 5+45s, the left 10s netperf data are not need, we force to kill them. but if it takes 15s on all session start, 15s+45s =60s, if uses ignore_status=false, here calls error is wrong, so ignore_status=true is better